### PR TITLE
[mui-datatables] add property fixedHeaderOptions

### PR DESCRIPTION
**Pull request by alexcberk**
Please fill in this template.

[ x] Use a meaningful title for the pull request. Include the name of the package modified.
[x ] Test the change in your own code. (Compile and run.)
[ x] Add or edit tests to reflect the change. (Run with npm test.)
[x ] Follow the advice from the readme.
[x ] Avoid common mistakes.
[ x] Run npm run lint package-name (or tsc if no tslint.json is present).
Select one of these and delete the others:

If changing an existing definition:

[x ] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/gregnb/mui-datatables
source: https://github.com/gregnb/mui-datatables/blob/master/src/MUIDataTable.js#L171

### CODE REVIEW

**Comment by Favna** [comment]
@sandersn Travis says the comment for minimum TS should be `// Minimum TypeScript Version: 3.1` but the README of the repo still says `// TypeScript Version: 3.1`, which this package has right now. Has the syntax changed without being backwards compatible? If so, that's going to cause testing errors for probably a LOT of packages in this repo unless they are all updated accordingly (a "find and replace all" might even suffice).

File: types/mui-datatables/index.d.ts [hunk 1]
```diff
@@ -194,6 +194,10 @@ export interface MUIDataTableOptions {
     filter?: boolean;
     filterType?: FilterType;
     fixedHeader?: boolean;
+    fixedHeaderOptions?: {
```
Comment (peterblazejewicz) [inline types/mui-datatables/index.d.ts]: > [ x] Add or edit tests to reflect the change. (Run with npm test.)

Can we amend the `mui-datatables-tests.tsx` to cover that change?
Just like in:
https://github.com/gregnb/mui-datatables/blob/8188cafd37e6ed76876c8129b5e928b18c73ba4d/examples/fixed-header/index.js#L82-L85
so,
```tsx
        sortFilterList: false,
        fixedHeaderOptions: {
            xAxis: false,
            yAxis: true
        },
        customRowRender: (data, dataIndex, rowIndex) => {
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM
@acberk  thanks for the contribution!

File: types/mui-datatables/index.d.ts (hunk 1 above)
Comment (acberk) [reply types/mui-datatables/index.d.ts]: will do

**Review by favna — APPROVED** [review]

**Pull request merged**